apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/familieforhold/barnepensjon/BarneListe.tsx

```diff
@@ -0,0 +1,102 @@
+import { Child } from '@navikt/ds-icons'
+import { Heading, Link, Table } from '@navikt/ds-react'
+import { IFamilieforhold, IPdlPerson } from '~shared/types/Person'
+import styled from 'styled-components'
+import { IAdresse } from '~shared/types/IAdresse'
+import { differenceInYears, format, parse } from 'date-fns'
+import { CopyToClipboard } from '@navikt/ds-react-internal'
+import { DatoFormat, formaterFnr } from '~utils/formattering'
+import { FlexHeader, IconWrapper, TableWrapper } from '~components/behandling/soeknadsoversikt/familieforhold/styled'
+import { IconSize } from '~shared/types/Icon'
+
+const FnrWrapper = styled.div`
+  display: flex;
+`
+
+type Props = {
+  familieforhold: IFamilieforhold
+}
+
+export const BarneListe = ({ familieforhold }: Props) => {
+  const barneListe = familieforhold.avdoede?.opplysning.avdoedesBarn ?? []
+
+  return (
+    <div>
+      <FlexHeader>
+        <IconWrapper>
+          <Child fontSize={IconSize.DEFAULT} />
+        </IconWrapper>
+        <Heading size={'small'} level={'3'}>
+          Søsken(avdødes barn)
```

Comment:
Mangler mellomrom før parantes 🤓

Reply:
Grunnen til at det så rart ut på gammel er det samme som for den som på ny. Nemlig at man kun har valgt å vise gjenlevende hvis den ikke er oppfylt.. Hvorfor?
Fjerner.

Reply:
bc63b2c011a91acb9a74310d5b806bfb68987f70